docs/analysis_modules.md

```diff
@@ -7,6 +7,50 @@ social:
 
 ## Plots
 
+### Line Plot
+
+<div class="clear" markdown>
+
+![Image title](assets/images/analysis_modules/plots/line_plot.svg){ align=right loading=lazy width="50%"}
+
+Line plots are particularly good for visualizing sequences that resemble time-based data, such as:
```

Comment:
I'd change this line to "Line plots are particularly good for visualizing sequences that are ordered or sequential, but not necessarily categorical, such as: